### ISSUE TITLE
Adding captions

### ISSUE THREAD

**Issue by ndwl** [kthornbloom/Smoothproducts#11]
Is there a way to captions under the thumbnails and in the lightbox?

**Comment by kthornbloom** [comment]
No, there currently isn't an option for captions. I could add captions in the future for the main image, however I don't see how they'd fit with the thumbnails. What did you have in mind?

**Comment by ndwl** [comment]
Hi Kevin,

I was planning to use it to show fabric swatches, so have the requirement to display the colour reference  beneath each of the thumbnails (and also under the image when viewed in a lightbox).

Captions aside, every other aspect has been a pleasure to use.

Regards - Nick


Nick Loat  MA(RCA)
Loat Davies

-----------------------------------
The Old Rectory Studio
Bourton Road
Moreton-in-Marsh
Gloucestershire
GL56 0BG
-----------------------------------
T:  +44 (0) 1608 812883
F:  +44 (0) 1608 812881
M: +44 (0) 7970 024271
-----------------------------------
Company no. 3069120
Vat no. 628 4989 83
-----------------------------------

**Comment by kthornbloom** [comment]
Glad it's working well for you! I'll see what I can do about some captions the next time I have some free time.

**Comment by ndwl** [comment]
That would be brilliant, many thanks

Regards - Nick


Nick Loat  MA(RCA)
Loat Davies

-----------------------------------
The Old Rectory Studio
Bourton Road
Moreton-in-Marsh
Gloucestershire
GL56 0BG
-----------------------------------
T:  +44 (0) 1608 812883
F:  +44 (0) 1608 812881
M: +44 (0) 7970 024271
-----------------------------------
Company no. 3069120
Vat no. 628 4989 83
-----------------------------------
